lib/ssh_scan/api.rb

```diff
@@ -104,6 +104,13 @@ class API < Sinatra::Base
           "#{params[:target]}:#{params[:port] ? params[:port] : "22"}"
         options[:policy_file] = options[:policy]
         options[:uuid] = SecureRandom.uuid
+        settings.count_req += 1
+        if (Time.now - settings.curr_time) > 60.0
+          settings.requests_per_min = 0
+          settings.curr_time = Time.now
+        else
+          settings.requests_per_min += 1
+        end
```

Comment:
I'm going to send you a PR as a suggestion or start to make this a bit more clear.